genesis/vis/visualizer.py

```diff
@@ -127,17 +128,36 @@ def destroy(self):
         self.viewer_lock = None
         self._renderer = None
 
-    def add_camera(self, res, pos, lookat, up, model, fov, aperture, focus_dist, GUI, spp, denoise, env_idx):
-        cam_idx = len(self._cameras)
+    def add_camera(self, res, pos, lookat, up, model, fov, aperture, focus_dist, GUI, spp, denoise, env_idx, debug):
+        cam_idx = len(self._cameras) if not debug else len(self._debug_cameras)
         camera = Camera(
-            self, cam_idx, model, res, pos, lookat, up, fov, aperture, focus_dist, GUI, spp, denoise, env_idx=env_idx
+            self,
+            cam_idx,
+            model,
+            res,
+            pos,
+            lookat,
+            up,
+            fov,
+            aperture,
+            focus_dist,
+            GUI,
+            spp,
+            denoise,
+            env_idx=0 if debug else env_idx,
+            debug=debug,
         )
-        self._cameras.append(camera)
+        if debug:
+            self._debug_cameras.append(camera)
+        else:
+            self._cameras.append(camera)
```

Comment:
I'm not a great fan of managing cameras and "debug cameras" separately. They are inherently the same things. Just the debug camera are not considered real sensors but only used to monitor the progress of the agent / debugging. Having `camera` getting hiding debug camera by default, and keeping them accessible via `_camera` private attribute sounds sufficient for me.